Description: Remove rmpv in test as it's not depended on
Last-Update: 2022-09-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/round.rs
+++ b/tests/round.rs
@@ -686,12 +686,10 @@
         Ok(t) => t,
         Err(e) => {
             panic!(
-                "Does not deserialize: {}\nConfig: {}\nSerialized {:?}\nGot {:?}\n",
+                "Does not deserialize: {}\nConfig: {}\nSerialized {:?}\n",
                 e,
                 desc,
                 val,
-                rmpv::decode::value::read_value(&mut serialized.as_slice())
-                    .expect("rmp didn't serialize correctly at all")
             );
         }
     };
